src/freenas/etc/ix.rc.d/ix_sshd_save_keys

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+#
+# $FreeBSD$
+#
+
+# PROVIDE: ix_sshd_save_keys
+# REQUIRE: openssh
+
+#
+# Save any generated keys by /usr/local/etc/rc.d/sshd into the config db.
+#
+
+. /etc/rc.freenas
+
+save_keys()
+{
+    LD_LIBRARY_PATH=/usr/local/lib /usr/local/bin/midclt call ssh.save_keys > /dev/null
```

Comment:
Can you remind me how this regressed? I can remember seeing this bug already which made us reroll for 11.3?

Didnt we already had routine to save ssh keys?

Reply:
In 11.3 we had a different bug: 70b80576736fbdbfbf36959505642d25a78f673b

I removed this file when I was making services cross-platform, thinking that saving keys after we start SSH service with middleware covers all cases. I was wrong: when we import config without pwenc secret but with SSH service autostart enabled, middleware is not aware of service startup and there is nothing to trigger saving keys.